batchflow/models/tf/unet.py

```diff
@@ -84,13 +79,10 @@ def build_config(self, names=None):
         initial_block_filters = config.get('initial_block/filters')
         f_0 = initial_block_filters[-1] if isinstance(initial_block_filters, (list, tuple)) else initial_block_filters
 
-        if config.get('body/encoder/blocks/filters') is None:
+        if config.get('body/encoder/blocks/filters') == [128, 256, 512, 1024] and num_stages != 4:
```

Comment:
That's not the best way to do this.
Perhaps you can get `default_config` here and compare it to the config passed to asses parameters changed.